packages/mobile/src/components/payment-method/CardSelectionButton.tsx

```diff
@@ -2,7 +2,7 @@ import { useCallback } from 'react'
 
 import type { PurchaseVendor } from '@audius/common'
 import { modalsActions } from '@audius/common'
-import { TouchableOpacity } from 'react-native-gesture-handler'
+import { TouchableOpacity } from 'react-native'
```

Comment:
I can no longer remember which one of these does and does not work on various platforms 😅

Reply:
`react-native-gesture-handler` = bad